build.sbt

```diff
@@ -41,9 +41,9 @@ addCommandAlias("fmt", "scalafmtAll")
 val testcontainersVersion = "0.41.4"
 val circeVersion = "0.14.10"
 
-lazy val root = project
+lazy val magnumRoot = project
```

Comment:
I wonder if the `root` name isn't necessary for sbt to understand that it's your root module 🤔